wrappers/csharp/cs-tutorial-8-D400-on-chip-calibration/Window.xaml.cs

```diff
@@ -57,13 +57,14 @@ public CaptureWindow()
                 using (var ctx = new Context())
                 {
                     var devices = ctx.QueryDevices();
-                    if ( 0 == devices.Count )
+                    var dev = devices[0];
```

Comment:
Won't this throw/crash if there aren't any?

Reply:
No, this is checked and verified internally in the next clause